src/features/Instructors/InstructorsFilters/index.jsx

```diff
@@ -42,12 +44,14 @@ const InstructorsFilters = ({ resetPagination, isAssignSection }) => {
 
   const [courseOptions, setCourseOptions] = useState([]);
 
-  const { formState, handleInputChange, resetFormState } = useFormInput(initialFilterState);
+  const {
+    formState, handleInputChange, resetFormState, touchedFields,
+  } = useFormInput(initialFilterState);
 
-  const disableSubmitButtons = formState.active === null
-  && !formState.instructorEmail
-  && !formState.instructorName
-  && formState.courseSelected === null;
+  const disableSubmitButtons = !touchedFields?.active
```

Comment:
Why is necessary the `?` here?

Reply:
touchedFields is by default an empty object, only will add the fields name when is touched so won't have all time the active attribute